app/models/manageiq/providers/hawkular/inventory/persister/middleware_manager.rb

```diff
@@ -0,0 +1,44 @@
+module ManageIQ::Providers
+  class Hawkular::Inventory::Persister::MiddlewareManager < ManagerRefresh::Inventory::Persister
+    include ManagerRefresh::Inventory::Core
+
+    COMMON_ATTRIBUTES = %i(name ems_ref nativeid properties feed).freeze
+
+    has_inventory(
+      :model_class                 => provider_module::MiddlewareManager::MiddlewareDomain,
```

Comment:
once you have a working refresh, could you move those definitions to `app/manager_refresh/inventory/core.rb` and `app/manager_refresh/inventory/middleware_manager.rb` ?